pkg/operator/controller/ingress/deployment.go

```diff
@@ -386,6 +386,7 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 		// problems or firewall restrictions.
 		deployment.Spec.Template.Spec.Containers[0].LivenessProbe.Handler.HTTPGet.Host = "localhost"
 		deployment.Spec.Template.Spec.Containers[0].ReadinessProbe.Handler.HTTPGet.Host = "localhost"
+		deployment.Spec.Template.Spec.DNSPolicy = "ClusterFirstWithHostNet"
```

Comment:
```suggestion
		deployment.Spec.Template.Spec.DNSPolicy = corev1.DNSClusterFirstWithHostNet
```
Probably better if we just use the constant directly from the API definitions, right?